example/main.tf

```diff
@@ -85,6 +85,38 @@ resource "google_compute_subnetwork" "vpc_subnetwork" {
   ]
 }
 
+resource "google_compute_router" "router" {
+  name    = "${var.router_name}"
+  network = "${google_compute_network.vpc_network.self_link}"
+
+  bgp {
+    asn = 64514
```

Comment:
Comment here? (maybe I just need to read the official docs, but not sure what this does..)

Reply:
I've now removed BGP stuff as it's not required for Cloud NAT.